lifetimes/datasets/__init__.py

```diff
@@ -43,3 +45,23 @@ def load_transaction_data(**kwargs):
 
     """
     return load_dataset('example_transactions.csv', **kwargs)
+
+
+def load_transaction_data_with_monetary_value(**kwargs):
+    transactions = pd.read_csv(
+        resource_filename(
+            'lifetimes', 'datasets/CDNOW_master.txt'
+        ),
+        sep='[\t|\s]+',
+        engine='python', **kwargs
+    )
+
+    transactions['date'] = pd.to_datetime(transactions['date'], format='%Y%m%d')
```

Comment:
This can be done in the call to `summary_data_from_transaction_data` by setting `datetime_format='%Y%m%d'`